specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2021-01-01/ResourceHealth.json

```diff
@@ -1489,7 +1491,7 @@
         }
       }
     },
-    "PotentialOutageErrorResponse": {
+    "WrappedErrorResponse": {
```

Comment:
Is not part of this PR but what is the format of ErrorResponse -> details? Is a JSON string?
The best way to format a detail is to be an object, you can simplify this definition by adding a reference to [this](https://github.com/Azure/azure-rest-api-specs/blob/8afb86737a4e792aa5d90280461e3cca4a2a9f52/specification/common-types/resource-management/v2/types.json#L313) common definition. Here's an [example](https://github.com/Azure/azure-rest-api-specs/blob/8afb86737a4e792aa5d90280461e3cca4a2a9f52/specification/resources/resource-manager/Microsoft.Resources/stable/2020-10-01/deploymentScripts.json#L884).

Reply:
Thanks for the comment, in previous version the "errorresponse" no longer meet the requirement, so it needs to be wrapped in a object. I am referring to the "errorresponse" in the wrappederrorresponse.

Reply:
Yeah, I get that part, but your formatting seems to be incorrect and it caused difficulties for customers to view the error details, that's why I was asking if your "details" property from ErrorResponse which is referenced in your WrappedErrorResponse is a JSON string, if not and just a plain text then what you have is correct, otherwise my recommendation is to adhere to the standard, see [here](https://github.com/Azure/azure-resource-manager-rpc/blob/e64480d4f830bd3ac18286e4ee1b2769ca7283d4/v1.0/common-api-details.md#error-response-content).

Reply:
ack